data/sles4sap/qe_sap_deployment/trento_azure.yaml

```diff
@@ -32,6 +32,7 @@ ansible:
     secondary_site: 'miky'
   create:
     - registration.yaml -e reg_code='%SCC_REGCODE_SLES4SAP%' -e email_address='testing@suse.com'
+    - fully-patch-system.yaml
```

Comment:
I'm not sure Trento want this

Reply:
Not sure too, but from the "Description" of this ticket: https://jira.suse.com/browse/TEAM-8437 I think I need to apply full patch for as many as test cases.

Reply:
I think we should fully patch in trento scenario.